AsyncDisplayKit/ASMapNode.mm

```diff
@@ -17,70 +17,80 @@ @interface ASMapNode()
   MKMapSnapshotter *_snapshotter;
   MKMapSnapshotOptions *_options;
   NSArray *_annotations;
-  ASDisplayNode *_mapNode;
```

Comment:
This is significantly better - thank you for all the changes, especially removing / limiting state!